util/predictoor/predictoor_testutil.py

```diff
@@ -0,0 +1,62 @@
+import random
+from typing import List
+from enforce_typing import enforce_types
+
+
+@enforce_types
+def create_mock_responses(n: int):
+    _statuses = ["Pending", "Paying", "Canceled"]
+    _weights = [1, 4, 1]
+    _count = 100
+
+    users = [f"0x{i}" for i in range(_count)]
+    stats = {user: {"total": 0, "correct": 0} for user in users}
+
+    def generate_responses(n: int):
+        responses = []
+        for _ in range(n):
+            payouts = [1.0 if random.random() > 0.4 else 0.0 for _ in range(_count)]
```

Comment:
why not math.floor? floor(random.random) should give the exact same result as the if statement